src/converter.js

```diff
@@ -77,10 +78,14 @@ var getPayload = function(swagger, path, method) {
       if (typeof param.in !== 'undefined' && param.in.toLowerCase() === 'body' &&
         typeof param.schema !== 'undefined') {
         try {
-          const sample = OpenAPISampler.sample(param.schema, { skipReadOnly: true }, swagger)
+          const sample = OpenAPISampler.sample(param.schema, { skipReadOnly: true }, swagger);
+          const paramContentType = swagger.consumes && swagger.consumes.length ?
```

Comment:
First you should handle `consumes` on param and only then should handle global definition